### PR TITLE
feat: piece store adds ldfs storage type

### PR DESCRIPTION
**Pull request by SunskyF**
### Description

Piece store adds ldfs object storage
Note!!!
Not tested yet, just a draft

### Rationale

Support a new object storage minio

### Example

add an example CLI or API response...

### Changes

Notable changes: 
* add each change in a bullet point here
* ...

### CODE REVIEW

File: model/piecestore/model.go [hunk 1]
```diff
@@ -41,6 +43,13 @@ const (
 	MinioSecretKey = "MINIO_SECRET_KEY"
 	// MinioSessionToken defines env variable name for minio session token
 	MinioSessionToken = "MINIO_SESSION_TOKEN"
+
+	// LdfsAccessKey defines env variable name for ldfs access key
+	LdfsAccessKey = "LDFS_ACCESS_KEY"
```
Comment (outan) [inline model/piecestore/model.go:48]: @SunskyF 
这个地方和nake聊过了。
- 这次的case的场景更关注性能，如果用AK/SK的话，性能上会有一些损失；
- 安全方面，调用方如ECS或ACK等是和Lindorm在一个VPC中，所以准备通过加白来授权访问的权限。

所以不需要这些AK/SK相关的内容。

Comment (SunskyF) [reply model/piecestore/model.go]: 已去除aksk相关代码

**Review by actioncli — APPROVED** [review]

**Review by joeylichang — APPROVED** [review]

**Comment by will-2012** [comment]
The PR should be submitted to the develop branch, not the master branch.

**Comment by SunskyF** [comment]
Sorry, I don't know how to modify the branch name while keeping the pull request.
So I open a new pr here  https://github.com/bnb-chain/greenfield-storage-provider/pull/545, please review this new pr.

**Pull request closed without merging**